Formula/p/prjtrellis.rb

```diff
@@ -0,0 +1,47 @@
+class Prjtrellis < Formula
+  desc "Documenting the Lattice ECP5 bit-stream format"
+  homepage "https://github.com/YosysHQ/prjtrellis"
+  url "https://github.com/YosysHQ/prjtrellis/archive/refs/tags/1.4.tar.gz"
+  sha256 "46fe9d98676953e0cccf1d6332755d217a0861e420f1a12dabfda74d81ccc147"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on "boost-python3"
+  depends_on "python@3.12"
+
+  resource "prjtrellis-db" do
+    url "https://github.com/YosysHQ/prjtrellis/releases/download/1.4/prjtrellis-db-1.4.zip"
+    sha256 "4f8a8a5344f85c628fb3ba3862476058c80bcb8ffb3604c5cca84fede11ff9f0"
+  end
+
+  def install
+    (buildpath/"database").install resource("prjtrellis-db")
+
+    system "cmake", "-S", "libtrellis", "-B", "libtrellis",
+                    "-DCURRENT_GIT_VERSION=1.4", *std_cmake_args
```

Comment:
Looks like this is still hardcoding version number?

Reply:
I guess I missed my own comment. let me update it